optuna/study/study.py

```diff
@@ -569,6 +633,9 @@ def df(x):
                 :class:`~optuna.trial.TrialState.PRUNED`.
                 ``trial`` is a trial number but no
                 trial exists with that number.
+
+        Returns:
+            A :class:`~optuna.trial.FrozenTrial`.
```

Comment:
How about some elaboration.
```suggestion
            A :class:`~optuna.trial.FrozenTrial` representing the resulting trial.
```